<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataManageUI</class>
 <widget class="QWidget" name="dataManageUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenCV Face Recognition System - DataManage</string>
  </property>
  <widget class="QGroupBox" name="dbManageGroupBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>341</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>数据管理</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QGroupBox" name="dbStatusGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>数据库状态</string>
    </property>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="dbUserCountHorizontalLayout">
      <item>
       <widget class="QLabel" name="dbUserCountLabel">
        <property name="text">
         <string>数据库已存人脸样本数：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="dbUserCountLcdNum">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="initDbButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>初始化数据库</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="trainGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>训练数据</string>
    </property>
    <widget class="QGroupBox" name="imgPreprocessGroupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>281</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>图像预处理</string>
     </property>
     <widget class="QCheckBox" name="equalizeHistCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>直方图均衡化</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="trainButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>数据库未完成初始化，请检查</string>
     </property>
     <property name="text">
      <string>开始训练</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="logGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>系统日志</string>
    </property>
    <widget class="QTextEdit" name="logTextEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>321</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="faceDbGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>人脸库</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QGroupBox" name="deleteUserGroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>561</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>删除用户</string>
    </property>
    <widget class="QGroupBox" name="queryUserGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>学号查询</string>
     </property>
     <widget class="QLabel" name="queryResultLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="queryUserLineEdit">
         <property name="placeholderText">
          <string>请输入12位工号，仅支持精确查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="queryUserButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>数据库未完成初始化，请检查</string>
         </property>
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="queryResultGroupBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>301</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>查询结果</string>
     </property>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>281</width>
        <height>87</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="infoHorizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="keyVerticalLayout">
         <item>
          <widget class="QLabel" name="stuIDLabel">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>工号：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cnNameLabel">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>姓名：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="faceIDLabel">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>Face ID：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="valueVerticalLayout">
         <item>
          <widget class="QLineEdit" name="stuIDLineEdit">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cnNameLineEdit">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="faceIDLineEdit">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="deleteUserButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>281</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>删除记录</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="dbVisualGroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>561</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>321</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>工号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Face ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>汉语拼音</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>注册时间</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="tipLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>320</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>注：Face ID为 -1 说明该用户的人脸数据未被训练</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
